stacs/clms_catalog.json

```diff
@@ -0,0 +1,31 @@
+{
+  "id": "clms_catalog",
+  "type": "Catalog",
+  "title": "CLMS Catalog",
+  "stac_version": "1.0.0",
+  "description": "This catalog is a simple demonstration of an example catalog that is used to organize a hierarchy of collections and their items.",
+  "links": [
+    {
+      "rel": "root",
+      "href": "https://github.com/eea/clms-stac/stacs/clms_catalog.json",
+      "type": "application/json"
```

Comment:
is not necessary a title here? like root catalog or something like that

Reply:
[Title](https://github.com/radiantearth/stac-spec/blob/master/catalog-spec/catalog-spec.md#link-object) is not required, but we can add it. See the [Sentinel-2 example](https://github.com/stactools-packages/sentinel2/blob/main/examples/catalog.json#L10)